client/src/components/CardItem/CardItem.module.scss

```diff
@@ -0,0 +1,76 @@
+.cardItem {
```

Comment:
Transfer all styled needed (except colors and font-sizes) to your js file and delete SCSS file.